sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/AppController.java

```diff
@@ -92,4 +92,17 @@
             return Result.ofFail(1, "remove failed");
         }
     }
+
+    @RequestMapping(value = "/getVersion")
```

Comment:
The `AppController` is used for managing applications. Maybe we can add a new `VersionController` then add the API (GET `/version`) here.

Reply:
+1, add a separate Controller is more independent and clear, easy to find.